meta/recipes-devtools/opkg/opkg-keyrings_1.0.bb

```diff
@@ -2,6 +2,8 @@ SUMMARY = "Keyrings for verifying opkg packages and feeds"
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
 
+PACKAGE_ARCH = "all"
```

Comment:
I notice some recipes using `inherit allarch` which also seems related. Can you check if that needs to be added in addition to this or in place of this?

Reply:
You're right, I just need to add `inherit allarch` in place of this.